dev-tools/iota-data-ingestion/README.md

```diff
@@ -57,27 +57,31 @@ tasks:
 
 ## Usage
 
-### Build the Docker Image
+#### 1. Build the required image
 
-Build a fresh Docker image without using cached layers:
+```shell
+pushd <iota project directory>/docker/iota-data-ingestion && ./build.sh && popd
+```
 
-```bash
-docker compose build --no-cache
+#### 2. CD into the iota-data-ingestion directory
+
+```shell
+cd <iota project directory>/dev-tools/iota-data-ingestion
 ```
 
-### Start the Service
+### 3. Start the Service
 
 Run the container in detached mode:
 
-```bash
+```shell
 docker compose up -d
```

Comment:
Not introduced by this PR, but this assumes that the user already has a `.env` file in that directory.

What values should the `.env` contain ? can we add this here in the README or a link to other docs if it's already documented ?
Also, the container seems to start fine with an empty `.env`, is that expected ?

And also, same as in my other comment, we could make this work from any directory:
```suggestion
docker compose -f "$(git rev-parse --show-toplevel)/dev-tools/iota-data-ingestion/docker-compose.yaml" up -d
```
so the reader can copy-paste the command

Reply:
my bad, I must have scrolled right past the docs for env vars.
And no problem for keeping the shorter docker-compose command if that seems easier